Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM ubuntu:20.04
+
+# Enviourment needed to install cmake.
+ENV TZ=US/Pacific
+RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
+
+RUN apt-get update && apt-get install -y && apt-get install -y apt-utils \
```

Comment:
New Ubuntu's prefer `apt` over `apt-get`:
https://itsfoss.com/apt-vs-apt-get-difference/

Reply:
SCRIPT USAGE
       The apt(8) commandline is designed as a end-user tool and it may change
       the output between versions. While it tries to not break backward
       compatibility there is no guarantee for it either. All features of
       apt(8) are available in apt-cache(8) and apt-get(8) via APT options.
       Please prefer using these commands in your scripts.